hw2/04-background.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>04 - Background</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+    <script defer src="04-background.js"></script>
+  </head>
+  <body style="background-color: blueviolet;">
+    <main class="bg-light mx-auto w-50 mt-5 rounded p-3">
```

Comment:
Looks good. The only thing is to set a smaller width to match the picture from the homework doc's.

Reply:
When background color is white, the border outside the text is not visible. adding broder class of bootstrap can resolve this problem. Verify the accesibility using WAVE tool to avoid any alerts.